src/pages/library/Datasets.test.js

```diff
@@ -38,4 +44,22 @@ describe('Datasets', () => {
     // We can test if the new catalog is shown by checking for filters on the page.
     expect(queryByText('Filters')).toBeTruthy()
   })
+
+  it('only shows RadX datasets when in RadX', () => {
+    getEnabledBrand.mockReturnValue(brands.radX)
+    expect(filterAndNormalizeDatasets({
+      result: [
+        { 'dct:title': 'radx-up', 'TerraDCAT_ap:hasDataCollection': [{ 'dct:title': 'RADx-UP' }] }, { 'dct:title': 'not-radx' }
+      ]
+    }).length).toBe(1)
```

Comment:
Should this test that the one item is the RadX dataset?

```suggestion
    const displayedDatasets = filterAndNormalizeDatasets({
      result: [
        { 'dct:title': 'radx-up', 'TerraDCAT_ap:hasDataCollection': [{ 'dct:title': 'RADx-UP' }] }, { 'dct:title': 'not-radx' }
      ]
    })
    
    expect(displayedDatasets.length).toBe(1)
    expect(displayedDatasets[0].title).toBe('radx-up')
```

Reply:
Or...

```suggestion
    expect(filterAndNormalizeDatasets({
      result: [
        { 'dct:title': 'radx-up', 'TerraDCAT_ap:hasDataCollection': [{ 'dct:title': 'RADx-UP' }] }, { 'dct:title': 'not-radx' }
      ]
    })).toEqual([expect.objectContaining({ 'dct:title': 'radx-up' })])
```